p2panda-rs/src/operation/operation.rs

```diff
@@ -455,6 +456,20 @@ impl From<&OperationEncoded> for Operation {
     }
 }
 
+impl PartialEq for Operation {
```

Comment:
I think this is an important difference between `Operation` and `OperationWithMeta` that should be highlighted in their descriptions. Would you mind if I added something there?

Reply:
You mean in the `PartialEq` implementation or in general for the structs? If its the latter you might be faster with making a new PR for the comments :smiley_cat:

Reply:
I think it would be better to have the same PartialEq implementation for both `Operation` and `OperationWithMeta`, or..?

Reply:
Why do you think so? Which one should it be?

Reply:
Aha, we just chatted, two `OperationWithMeta` with the same field values but different authors are not equal, whereas two `Operation` with the same field values are always equal, so the `PartialEq` should be obvs different. Nice :+1: